activerecord/lib/active_record/readonly_attributes.rb

```diff
@@ -15,7 +15,8 @@ module ClassMethods
       # Attributes listed as readonly will be used to create a new record but update operations will
       # ignore these fields.
       #
-      # You can assign a new value to a readonly attribute, but it will be ignored when the record is updated.
+      # If `config.active_record.raise_on_assign_to_attr_readonly` is true, then if you assign a new
+      # value to a readonly attribute it will raise an error. This is the default.
```

Comment:
I think we can improve this section a bit more by first describing the default behaviour, and then mentioning the ability to override it:
```suggestion
      # Assigning a new value to a readonly attribute raises an error by default.
      # By setting `config.active_record.raise_on_assign_to_attr_readonly` to `false` it will not
      # raise and ignore the attribute instead.
```